build-a-ruby-on-rails-app-with-tidb.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Power Up Your Rails Apps with a NewSQL Database
+author: ['Matt Wang']
+date: 2020-09-30
+summary: This post helps Ruby on Rails developers get started with TiDB and use it as the backend storage layer of Rails applications.
+tags: ['Tutorial']
+categories: ['Community']
+image: /images/blog/build-rails-apps-with-tidb-banner.png
+---
+
+**Author:** [Matt Wang](https://github.com/hooopo) (Engineer at PingCAP, moderator of Ruby-China community)
+
+**Editor:** [Fendy Feng](https://github.com/septemberfd), Tom Dewan
+
+![Build a Rails App with TiDB](media/build-rails-apps-with-tidb-banner.png)
```

Comment:
```suggestion
![Build a Rails App with a NewSQL Database](media/build-rails-apps-with-a-newsql-database.png)
```
We can insert as many keywords as possible in image names and alt text for SEO purposes. Please change the image file name accordingly.